### PR TITLE
Update give-mailbox-permissions-to-another-user.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated more info on the note

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6962

File: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ There are a few different ways you can access a mailbox once you've been given a
 
 
 > [!NOTE]
-> **Read** and **Manage** permissions are called **Full Access** permission when granted in the Exchange admin center. Full Access permission does not grant **Send as** or **Send on Behalf**  permissions.
+> **Read** and **Manage** permissions are called **Full Access** permission when granted in the Exchange admin center. This permission allows the assigned user mailbox(es) to read as well as manage emails in the user mailbox on which the permission is assigned. Full Access permission does not grant **Send as** or **Send on Behalf**  permissions.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md]:
```suggestion
> **Read** and **Manage** permissions are called **Full Access** permission when granted in the Exchange admin center. This permission allows the assigned user mailbox to read as well as manage emails in the user mailbox on which the permission is assigned. Full Access permission does not grant **Send as** or **Send on behalf**  permissions.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**